app/controller/requests.rb

```diff
@@ -5,10 +5,18 @@ class Bnb < Sinatra::Base
   end
 
   post '/requests' do
-    booking = Booking.get(params[:host_request_id])
-    booking.status = 'Confirmed'
-    booking.save
-    flash.keep[:notice] = "Space request confirmed :)"
-    redirect('/requests')
+    if  params[:confirmed]
+      booking = Booking.get(params[:host_request_id])
+      booking.status = 'Confirmed'
```

Comment:
Consistency: Statuses need to be all lowercase or capitalized (pending is currently lowercase. confirmed is lowercase in the spaces.erb view filtering too)